frontend/src/employee-frontend/components/absences/AbsenceTable.tsx

```diff
@@ -109,32 +129,79 @@ const AbsenceTableRow = React.memo(function AbsenceTableRow({
           <DisabledCell />
         </td>
       ))}
+      {reservationEnabled && (
+        <>
+          <td>
+            <NumbersColumn>
+              {reservationTotalHours ? `${reservationTotalHours} h` : '-'}
+              <Gap size="xs" horizontal />
+              <WarningPlaceholder />
+            </NumbersColumn>
+          </td>
+          <td>
+            <NumbersColumn warning={showAttendanceWarning}>
+              {attendanceTotalHours ? `${attendanceTotalHours} h` : '-'}
+              <Gap size="xs" horizontal />
+              {showAttendanceWarning ? (
+                <FontAwesomeIcon
+                  icon={fasExclamationTriangle}
+                  size="1x"
+                  color={theme.colors.status.warning}
+                />
```

Comment:
I'm not a huge fan of these warning icons. So much clutter just to get everything justified in a limited space table column 😛 I know this is how it's designed, but I'd think a bolder font with a warning accent color would suffice.

Reply:
In this care the column headers are wide enough for the icons to not actually affect the layout. It's basically padding that also aligns the content regardless of its length.